servers/rendering/renderer_rd/shaders/skeleton.glsl

```diff
@@ -0,0 +1,199 @@
+#[compute]
+
+#version 450
+
+VERSION_DEFINES
+
+layout(local_size_x = 64, local_size_y = 1, local_size_z = 1) in;
+
+layout(set = 0, binding = 1, std430) buffer restrict writeonly DstVertexData {
```

Comment:
Is there a reason this isn't in binding 0?

If it runs I guess it is fine. But it seems sketchy.

Reply:
because push constant is at 0. Even if not actual part of the descriptor set, its a GLSL annoyance